herddb-core/src/main/java/herddb/codec/RecordSerializer.java

```diff
@@ -60,6 +60,29 @@
 
     private static final int INITIAL_BUFFER_SIZE = SystemProperties.getIntSystemProperty("herddb.serializer.initbufsize", 4 * 1024);
 
+    public static Object deserialize(Bytes data, int type) {
```

Comment:
this is a copy of the other serialize(byte[]), it is worth to cut and paste, in order to be able to skip copies